python/az/aro/azext_aro/custom.py

```diff
@@ -479,20 +480,24 @@ def aro_update(cmd,
             if client_id is not None:
                 oc_update.service_principal_profile.client_id = client_id
 
-    if platform_workload_identities is not None:
-        pwis = {}
-        for i in oc.platform_workload_identity_profile.platform_workload_identities:
-            pwis[i.operator_name] = openshiftcluster.PlatformWorkloadIdentity(
-                operator_name=i.operator_name,
-                resource_id=i.resource_id
-            )
+    if oc.platform_workload_identity_profile is not None:
+        if platform_workload_identities is not None or upgradeable_to is not None:
+            oc_update.platform_workload_identity_profile = openshiftcluster.PlatformWorkloadIdentityProfile()
 
-        for i in platform_workload_identities:
-            pwis[i.operator_name] = i
+        if platform_workload_identities is not None:
+            pwis = {}
+            for i in oc.platform_workload_identity_profile.platform_workload_identities:
+                pwis[i.operator_name] = openshiftcluster.PlatformWorkloadIdentity(
+                    operator_name=i.operator_name,
+                    resource_id=i.resource_id
+                )
 
-        oc_update.platform_workload_identity_profile = openshiftcluster.PlatformWorkloadIdentityProfile(
-            platform_workload_identities=list(pwis.values())
-        )
+            for i in platform_workload_identities:
+                pwis[i.operator_name] = i
+
+            oc_update.platform_workload_identity_profile.platform_workload_identities=list(pwis.values())
+        
+        oc_update.platform_workload_identity_profile.upgradeable_to = upgradeable_to 
```

Comment:
I think, we shouldn't be overwriting the upgradeable_to if it is `None`?

Reply:
The oc_update object is a patch object and is otherwise-empty, so we're overwriting None with None in this case.

Reply:
So, in that case, would be impacting the `upgradeable_to` in the doc or not? Just concerned, as the platform_workload_identity_profile is pointer type. Checking and confirming here.

Reply:
The field is being handled correctly at the backend, so keeping it none won't be causing the overwrite. Thanks!!